src/layouts/MainSection.js

```diff
@@ -1,6 +1,12 @@
+import MainButton from '../components/HallOfFameButton'
+
 const MainSection = () => {
   const mainSection = document.createElement('div');
   mainSection.classList.add('mainSection');
+
+  mainSection.appendChild(MainButton("h", "all of fame"));
```

Comment:
Czy to działa dobrze, jeżeli wywołując MainButton korzystasz w nim tylko z drugiego argumentu? "h" nie powinno się pojawić w tekście przez to.